Makefile

```diff
@@ -11,6 +11,9 @@ test-cpu: FORCE
 test-cuda: FORCE
 	pytest -v --device cuda --dtype all --cov=kornia test/ --flake8 --mypy
 
+test-mps: FORCE
+	pytest -v --device mps --dtype float32 -k "not (grad or exception or jit)"  test/
```

Comment:
we need to revisit how we tag tests because i'm not quite sure it will skip all the ones you marked

Reply:
The command I put here skips by the test name, so it will
https://stackoverflow.com/questions/47691512/exclude-a-list-of-tests-with-py-test-but-on-the-command-line